### ISSUE TITLE
margins out of control

### ISSUE THREAD

**Issue by barak** [mcnees/LaTeX-Graph-Paper#3]
This is a fantastic idea, thanks for writing it! I'm going to be using it a lot.

Noticed one problem, in trying to customize to different sorts of paper. Removing the `\usepackage[cm]{fullpage}`, or changing it to `\usepackage[in]{fullpage}` or even `\usepackage[margin=0cm]{geometry}` or `\usepackage[margin=5cm]{geometry}` makes no difference to the generated PDF file. And adding `documentclass[11pt,a4paper]{article}` just messes things up.

Basically, the graph-paper-drawing routines have the size of the area to fill hard coded, with things like `(8.1in,10.6in)`. These values should be computed from `\textheight` and `\textwidth`, and doing this would seem to require something like `\usepackage{calc}`.

**Comment by mcnees** [comment]
There's an initial commit of a package that works across different paper sizes. Lots of kinks still need to be ironed out (some patterns don't align properly on non-letter paper sizes yet, and we haven't added code to adjust margins for patterns that frame a grid) but we'll get there. You can open new issues for specific pattern / paper size bugs.

**Comment by barak** [comment]
Sweet!
